README.md

```diff
@@ -48,6 +51,50 @@ Each of the extensions / integrations has to be installed separately.
 | [`fuel-jackson`](./fuel-jackson) | _Deserialization_: [`Jackson`](https://github.com/FasterXML/jackson-module-kotlin)
 | [`fuel-moshi`](./fuel-moshi) | _Deserialization_: [`Moshi`](https://github.com/square/moshi)  |
 
+### Jitpack
+
+```kotlin
+repositories {
+  maven(url = "https://jitpack.io") {
+    name = "jitpack"
+  }
+}
+
+dependencies {
+  implementation(group = "com.github.kittinunf.fuel", name = "fuel", version = "-SNAPSHOT")
+  implementation(group = "com.github.kittinunf.fuel", name = "fuel-coroutines", version = "-SNAPSHOT")
+  implementation(group = "com.github.kittinunf.fuel", name = "fuel-kotlinx-serialization", version = "-SNAPSHOT")
+}
+```
+
+
+```kotlin
```

Comment:
this is a example of how to make the configuration more compact, maybe this should be mentioned explicitly